common/src/main/java/io/seata/common/util/BeanUtils.java

```diff
@@ -45,9 +45,9 @@ public static String beanToString(Object o) {
         for (Field field : fields) {
             Object val = null;
             try {
-                val = ReflectionUtil.getFieldValue(o, field.getName());
-            } catch (NoSuchFieldException e) {
-                LOGGER.warn(e.getMessage(), e);
+                val = ReflectionUtil.getFieldValue(o, field);
+            } catch (RuntimeException e) {
```

Comment:
IllegalArgumentException | SecurityException

Reply:
Currently, this method is only used for log printing, so I think all exceptions should be caught.

Reply:
Printing the error log is enough.